crates/vm/levm/src/vm.rs

```diff
@@ -709,14 +709,13 @@ impl VM {
             .and_then(|gas| gas.checked_add(refunded_gas))
             .ok_or(VMError::Internal(InternalError::UndefinedState(0)))?;
 
-        let gas_return_amount = self
+        let gwei_return_amount = self
```

Comment:
I think the value is measured in wei

Reply:
Yeah. With that changed we can merge

Reply:
Done in [ee33452](https://github.com/lambdaclass/ethrex/pull/1464/commits/ee33452a232dc510dd4332edb5df43fdf44dfc92)